hacking/env-setup.fish

```diff
@@ -49,15 +49,14 @@ set -gx ANSIBLE_LIBRARY $ANSIBLE_HOME/library
 
 # Generate egg_info so that pkg_resources works
 pushd $ANSIBLE_HOME
+if test -e $PREFIX_PYTHONPATH/ansible*.egg-info
+    rm -r $PREFIX_PYTHONPATH/ansible*.egg-info
+end
 if [ $QUIET ]
     python setup.py -q egg_info
 else
     python setup.py egg_info
 end
-if test -e $PREFIX_PYTHONPATH/ansible*.egg-info
-    rm -r $PREFIX_PYTHONPATH/ansible*.egg-info
-end
-mv ansible*egg-info $PREFIX_PYTHONPATH
```

Comment:
It was removed from `env-setup` so I was doing the same for `env-setup.fish`. Kudos for finding the actual code change that made the `mv` unnecessary.